docs/reference/actions/fresh.md

```
@@ -0,0 +1,86 @@
+# axon fresh
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)